aip/custom_org/0121.md

```diff
@@ -0,0 +1,95 @@
+---
```

Comment:
Test custom org aip... only front-matter changed

Reply:
This is in the wrong directory to actually be in the collection though, so the collection is probably not doing anything.

Reply:
I have tried 2-3 times to get collections to work well, with no luck. May you succeed where I failed. :-)